ios/Exponent/ExpoNotificationCenter/EXUserNotificationCenter.h

```diff
@@ -0,0 +1,30 @@
+//  Copyright © 2018 650 Industries. All rights reserved.
+
+#import <Foundation/Foundation.h>
+#import <UserNotifications/UserNotifications.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface EXUserNotificationCenter : NSObject
```

Comment:
Add a comment explaining that this class provides thread-safety for UNUserNotificationCenter on iOS 10 and 11. We don't want to add more methods to this class unless UNUserNotificationCenter adds those methods, and can remove the class when we drop support for iOS 10 and 11.